### PR TITLE
Fix flaky tests

### LINKED ISSUE #99: Fix flaky tests
Example flaky run: https://github.com/sfbrigade/compass/actions/runs/5182125681/attempts/1

### PR DESCRIPTION
**Pull request by codetheweb**
Uses [nextjs-server-modules](https://github.com/seamapi/nextjs-server-modules) to statically build Next.js routes.

I believe the issue was due to multiple Next.js dev servers mutating files in .next concurrently under certain circumstances.

Closes #99.

### CODE REVIEW

File: src/backend/routers/paras.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ export const paraProcedures = {
   getParaById: procedure
     .input(z.object({ user_id: z.string().uuid() }))
     .query(async (req) => {
-      console.log("req", req);
```
Comment (amantri) [inline src/backend/routers/paras.ts:9]: Did you want to leave this in there?

File: src/backend/tests/fixtures/get-test-server.ts [hunk 2]
```diff
@@ -41,24 +40,18 @@ export const getTestServer = async (
...
-  await app.prepare().then(() => {
-    server = createServer(async (req, res) => {
-      (req as unknown as { env: Env }).env = env;
-      await handle(req, res);
-    });
-    server.listen(appPort);
+  const server = await builtNextJsFixture({
+    port: appPort,
+    middlewares: [
+      (next) => (req, res) => {
```
Comment (amantri) [inline src/backend/tests/fixtures/get-test-server.ts:46]: 🐧 Add a comment indicating what is this doing or a link to documentation about how to use this JS fixture.

**Review by amantri — APPROVED** [review]

File: src/backend/routers/paras.ts (hunk 1 above)
Comment (tessathornberry) [reply src/backend/routers/paras.ts:9]: This is entirely due to our team leaving it there, sorry. Do you want to comment it out or delete it? It's no longer there in the newer PR for archivePara

File: src/backend/tests/fixtures/get-test-server.ts (hunk 2 above)
Comment (codetheweb) [reply src/backend/tests/fixtures/get-test-server.ts:47]: [a1f30cc](https://github.com/sfbrigade/compass/pull/104/commits/a1f30cc165934e58f74f3ba24aad1f97461873d7)

**Pull request merged**